promql/engine.go

```diff
@@ -677,7 +700,8 @@ func (ev *evaluator) rangeEval(f func([]Value, *EvalNodeHelper) Vector, exprs ..
 	for i, e := range exprs {
 		// Functions will take string arguments from the expressions, not the values.
 		if e != nil && e.Type() != ValueTypeString {
-			matrixes[i] = ev.eval(e).(Matrix)
+			ret := ev.eval(e)
```

Comment:
You don't need to change this

Reply:
:+1: sorry, leftover from when I modified the function signatures to return errors